skops/card/_model_card.py

```diff
@@ -373,6 +374,34 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_feature_importances(self, feature_importances, data) -> "Card":
+        """Visualize permutation importance.
```

Comment:
I think the docstring should also mention that a `feature_importances.png` is created.

Reply:
I edited it.